### PR TITLE
Adding doc and cleaning more modules

### PR DESCRIPTION
**Pull request by ja-che**

### CODE REVIEW

File: hidimstat/permutation_test.py [hunk 1]
```diff
@@ -11,9 +11,48 @@
...
-def permutation_test_cv(X, y, method='SVR', n_permutations=1000, C=None,
-                        normalize=False, random_state=None, n_jobs=1,
-                        verbose=1):
+def permutation_test_cv(X, y, n_permutations=1000, C=None,
+                        seed=0, n_jobs=1, verbose=1):
+    """Cross-validated permutation test shuffling the target
+
+    Parameters
+    -----------
+    X : ndarray or scipy.sparse matrix, (n_samples, n_features)
```
Comment (agramfort) [inline hidimstat/permutation_test.py:20]:
```suggestion
    X : ndarray or scipy.sparse matrix, shape (n_samples, n_features)
```

File: hidimstat/permutation_test.py [hunk 2]
```diff
@@ -11,9 +11,48 @@
...
+        Data.
+
+    y : ndarray, shape (n_samples,) or (n_samples, n_targets)
+        Target. Will be cast to X's dtype if necessary.
+
+    C : float or None, optional (default=None)
+        If None, the linear SVR regularization parameter is set by cross-val.
```
Comment (agramfort) [inline hidimstat/permutation_test.py:27]: with what CV generator? With what grid of Cs ?

File: hidimstat/permutation_test.py [hunk 3]
```diff
@@ -32,33 +71,63 @@ def permutation_test_cv(X, y, method='SVR', n_permutations=1000, C=None,
...
 
 
 def permutation_test(X, y, estimator, n_permutations=1000,
-                     normalize=False, random_state=None, n_jobs=1, verbose=1):
+                     seed=0, n_jobs=1, verbose=1):
+    """Permutation test shuffling the target
+
+    Parameters
+    -----------
+    X : ndarray or scipy.sparse matrix, (n_samples, n_features)
```
Comment (agramfort) [inline hidimstat/permutation_test.py:87]:
```suggestion
    X : ndarray or scipy.sparse matrix, shape (n_samples, n_features)
```

File: hidimstat/permutation_test.py [hunk 4]
```diff
@@ -69,17 +138,41 @@ def permutation_test(X, y, estimator, n_permutations=1000,
...
+def _shuffle(y, rng):
+    """Shuffle vector"""
+    indices = rng.permutation(len(y))
     return _safe_indexing(y, indices)
 
 
-def _step_down_max_T(stat, permutation_stats):
+def step_down_max_T(stat, permutation_stats):
+    """
+    Step-down maxT algorithm for computing adjusted p-values
```
Comment (agramfort) [inline hidimstat/permutation_test.py:154]:
```suggestion
    """Step-down maxT algorithm for computing adjusted p-values.
```

File: hidimstat/permutation_test.py [hunk 5]
```diff
@@ -69,17 +138,41 @@ def permutation_test(X, y, estimator, n_permutations=1000,
...
+    -------
+    pval_corr : array, shape (n_features,)
+        Adjusted p-values testing the null on the coefficients
+        of the parameter vector.
+
+    References
+    ----------
+    .. [1] Westfall, P. H., & Young, S. S. (1993).
+    Resampling-based multiple testing: Examples and methods for
+    p-value adjustment (Vol. 279). John Wiley & Sons.
```
Comment (agramfort) [inline hidimstat/permutation_test.py:174]:
```suggestion
            Resampling-based multiple testing: Examples and methods for
            p-value adjustment (Vol. 279). John Wiley & Sons.
```

File: hidimstat/standardized_svr.py [hunk 6]
```diff
@@ -5,27 +5,56 @@
 from sklearn.pipeline import Pipeline
 
 
-def standardized_svr(X, y, Cs=np.logspace(-7, 1, 9), return_C=False, n_jobs=1):
+def standardized_svr(X, y, Cs=None, return_C=False, n_jobs=1):
+    """Cross-validated SVR
+
+    Parameters
+    -----------
+    X : ndarray or scipy.sparse matrix, (n_samples, n_features)
```
Comment (agramfort) [inline hidimstat/standardized_svr.py:13]: shape

**Review by agramfort — APPROVED** [review]

File: hidimstat/standardized_svr.py [hunk 7]
```diff
@@ -5,27 +5,56 @@
...
+        Data.
+
+    y : ndarray, shape (n_samples,) or (n_samples, n_targets)
+        Target. Will be cast to X's dtype if necessary.
+
+    Cs : ndarray, default=None
+        List of Cs where to compute the models.
+        If None, Cs are set automatically.
```
Comment (bthirion) [inline hidimstat/standardized_svr.py:21]: default is  `np.logspace(-7, 1, 9)`

File: hidimstat/standardized_svr.py [hunk 8]
```diff
@@ -5,27 +5,56 @@
...
+
+    return_C : bool, default=False
```
Comment (bthirion) [inline hidimstat/standardized_svr.py:23]: I think we should avoid arguments that change function signature. It makes function harder to understand and debug.

**Review by bthirion — COMMENTED** [review]
Nice work !

File: hidimstat/standardized_svr.py [hunk 9]
```diff
@@ -5,30 +5,48 @@
...
+        Estimated parameter vector.
+
+    scale : ndarray, shape (n_features,)
+        Value of the standard deviation of the parameters.
+    """
 
     n_samples, n_features = X.shape
 
-    steps = [('SVR', LinearSVR())]
+    if Cs is None:
```
Comment (bthirion) [inline hidimstat/standardized_svr.py:37]: You probably don't need that, since there's a default Cs. Otherwise you should test it

**Review by bthirion — COMMENTED** [review]
LGTM otherwise.

**Pull request merged**